src/intree.js

```diff
@@ -36,6 +36,10 @@ function completeInTreeConfig(config, payload) {
     config.scopes = [
       `assume:repo:github.com/${ payload.organization }/${ payload.repository }:release`,
     ];
+  } else if (payload.details['event.type'] == 'tag') {
+    config.scopes = [
+      `assume:repo:github.com/${ payload.organization }/${ payload.repository }:release`,
```

Comment:
We want to treat tag events identical to release events. So i thought it would be appropriate to give them the same scopes as a release event. Is it a long process to add new scopes , if we want to do it differently ?

Reply:
It's as easy as just naming a different scope.  So this could be `:tag:<tagName>` instead of `:release`.